charts/rollout-operator/templates/webhook-role/validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.webhooks.enabled -}}
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    grafana.com/inject-rollout-operator-ca: "true"
+    grafana.com/namespace: {{.Release.Namespace}} 
+    {{- include "rollout-operator.labels" . | nindent 4 }}
+  name: no-downscale-{{.Release.Namespace}}
+webhooks:
+- name: no-downscale-{{.Release.Namespace}}.grafana.com
+  admissionReviewVersions: [v1]
+  clientConfig:
+    service:
+      name: {{ include "rollout-operator.fullname" .}}
+      namespace: {{.Release.Namespace}}
+      path: /admission/no-downscale
+      port: 443
+  {{- $failpol := (lookup "v1" "ValidatingWebhookConfiguration" "{{.Release.Namespace}}" "no-downscale-{{.Release.Namespace}}") -}}
+  {{- if empty $failpol }}
```

Comment:
I think it would be nice to have an explicit switch that would disable the failurePolicy, just in case something goes wrong here (like: the webhook is created, but the rollout-operator isn't updated so it's not updating its TLS certs)

Reply:
by disable, do you mean set `failurePolicy: Ignore`?
sorry i'm a bit out of the loop honestly this PR has been going on for a while 😄